src/main/java/lotto/model/LottoGenerator.java

```diff
@@ -0,0 +1,54 @@
+package lotto.model;
+
+import lotto.util.TypeConverter;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import static lotto.model.Lotto.CORRECT_LOTTO_NUMBERS_SIZE;
+
+public class LottoGenerator {
+    private static final String CAN_NOT_INPUT_NULL_MESSAGE = "공백은 입력 할 수 없습니다.";
+    private static final List<LottoNumber> numbers =
+            IntStream.rangeClosed(1, 45)
+                    .mapToObj(number -> new LottoNumber(number))
+                    .collect(Collectors.toList());
+
+    private LottoGenerator() {
+    }
+
+    public static Lotto makeManualLotto(String inputNumber) {
+        validateInputNumber(inputNumber);
+        Set<LottoNumber> lottoNumbers = TypeConverter.convertStringToLottoNumberSet(inputNumber);
+
+        return new Lotto(lottoNumbers);
+    }
+
+    private static void validateInputNumber(String inputNumber) {
+        if (inputNumber == null) {
+            throw new IllegalArgumentException(CAN_NOT_INPUT_NULL_MESSAGE);
+        }
+    }
+
+    public static BunchOfLotto makeBunchOfAutoLotto(int purchasedLottoCount) {
+        BunchOfLotto bunchOfLotto = new BunchOfLotto();
+
+        for (int i = 0; i < purchasedLottoCount; i++) {
+            bunchOfLotto.addLotto(LottoGenerator.makeAutoLotto());
+        }
+
+        return new BunchOfLotto(bunchOfLotto);
+    }
+
+    public static Lotto makeAutoLotto() {
+        Collections.shuffle(numbers);
+        Set<LottoNumber> lottoNumbers = new HashSet<>();
+
+        for (int i = 0; i < CORRECT_LOTTO_NUMBERS_SIZE; i++) {
+            lottoNumbers.add(numbers.get(i));
+        }
+
+        return new Lotto(lottoNumbers);
```

Comment:
```suggestion
        Set<LottoNumber> lottoNumbers = new HashSet<>(numbers.subList(0, 6));
        return new Lotto(lottoNumbers);
```
위와 같이 변경할 수도 있을 것 같아요 😁 (유용한 자바 API 참 많죠...?)